routes/properties.js

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+var Boom = require('boom');
+
+var knex = require('../connection.js');
+var rp = require('request-promise');
+var api = 'http://localhost:4000/';
+var queryWays = require('../services/query-ways');
+var flattenDeep = require('lodash.flattendeep');
+var groupBy = require('lodash.groupby')
+var includes = require('lodash.includes');
+var assign = require('lodash.assign');
+var reduce = require('lodash.reduce');
+Promise = require('bluebird');
+module.exports = [
+  /**
+   * @api {get} /ids Return all VProMMS ids in the database
+   *
+   * @apiSuccess {Array} ids IDs
+   *
+   * @apiExample {curl} Example Usage:
+   *    curl http://localhost:4000/ids
+   *
+   * @apiSuccessExample {json} Success-Response:
+   *  [1, 2, 3, 4]
+   */
+  {
+    method: 'GET',
+    path: '/properties',
+    handler: function (req, res) {
+      let desired;
+      if (req.query) {
+
+      }
+      rp(api + 'ids')
```

Comment:
Instead of make a request to the API itself, can we bring in the logic from the `/ids` route? This feels janky to me.